### PR TITLE
Add and Remove Functionality

### PR DESCRIPTION
**Pull request by Epaltechs**
### In this milestone, I :
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- Ensure all changes to the To-Do List is saved in local storage.

### CODE REVIEW

File: src/todos.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.index !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
```
Comment (andersonlebon) [inline src/todos.js]: - [ ] Kindly make sure When you remove tasks the last element also should be removed in the localstoreage. 


https://user-images.githubusercontent.com/65068771/166837082-209f76b1-1674-4ee4-9d51-f66d8db99bc8.mp4

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Epaltechs 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Epaltechs** [comment]
### In this commit, I:
- Modify the JavaScript file so that when one removes tasks, the last element also should be removed in the local storage.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Epaltechs, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**